.gitignore

```diff
@@ -88,3 +88,5 @@ pythonenv*/
 # JS dependencies
 mesa/visualization/templates/external/
 mesa/visualization/templates/js/external/
+*.pclprof
+*.pickle
```

Comment:
On line 2 `benchmarking/**/*.pickle` is already defined. Is it needed to ignore pickles everywhere?

Reply:
I thought, too, that you added them to .gitignore, but somehow it was not picked up on my machine. I will take a closer look.

I also have a few more minor changes coming due to the cell space comparison (see also https://github.com/projectmesa/mesa/issues/1900#issuecomment-1902643678)

Reply:
Maybe rebase your branch on main, it looks like it's lagging a bit behind.

Reply:
It should indeed have been `benchmarks/**/*.pickle`, good catch